elements/pfe-cta/src/pfe-cta.scss

```
@@ -77,13 +70,10 @@ $LOCAL-VARIABLES: map-deep-merge($LOCAL-VARIABLES, (
   font-weight: pfe-local(FontWeight);
   line-height: pfe-local(LineHeight);
   text-decoration: pfe-local(TextDecoration) !important;
-  
-  // cursor: pointer;
 
   :host([priority]) & {
-    --pf-c--FontSize: #{pfe-local(FontSize--priority)};
+    font-size: #{pfe-local(FontSize--priority)};
```

```suggestion
    font-size: pfe-local(FontSize--priority);
```

^ Just needs the interpolation removed (we only need that functions assigned to custom properties)